project/frontend/src/FindMatch/FindMatch.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class PreferenceForm extends React.Component {
+    render() {
```

Comment:
You need a return statement inside of your render - so after `render() {` put a `return(...all content in here)`